### ISSUE TITLE
Most recent (Feb 21 2024) patch is detected as virus by windows defender.

### ISSUE THREAD

**Issue by AtmaDarkwolf** [valinet/ExplorerPatcher#2894]
As topic. Am hoping just a mistake with code, will not update and wait on fix.(have never had it do this before, may be recent windows/Microsoft 'fuck you' to anyone attempting to make windows/file explorer useful)

**Comment by spafil** [comment]
Waiting for a "clean" update as Windows Defender got all hot and bothered with this one.

**Comment by pniaps** [comment]
Duplicated #2873

**Comment by Amrsatrio** [comment]
Duplicate of https://github.com/valinet/ExplorerPatcher/issues/2873
